src/makeRobot.js

```diff
@@ -39,6 +39,71 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    robotName: name,
+    robotWheels: wheels,
+    robotVersion: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.robotName}, `
+      + `chip version: ${this.robotVersion}, wheels: ${this.robotWheels}`;
+    },
+
+    get location() {
+      return `${this.robotName}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(value = 1) {
+      if (value <= 0) {
+        this.coords.y += 0;
+      } else {
+        this.coords.y += value;
+      }
+
+      return this;
+    },
+
+    goBack(value = 1) {
+      if (value <= 0) {
+        this.coords.y += 0;
+      } else {
+        this.coords.y -= value;
+      }
```

Comment:
you can make it shortest
```suggestion
      if (value > 0) {
        this.coords.y -= value;
      }
```